lib/rubocop/cop/rspec/feature_methods.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for consistent method usage in feature specs.
+      #
+      # @example
+      #   # bad
+      #   feature 'User logs in' do
+      #     given(:user) { User.new }
+      #
+      #     background do
+      #       visit new_session_path
+      #     end
+      #
+      #     scenario 'with OAuth' do
+      #       # ...
+      #     end
+      #   end
+      #
+      #   # good
+      #   describe 'User logs in' do
+      #     let(:user) { User.new }
+      #
+      #     before do
+      #       visit new_session_path
+      #     end
+      #
+      #     it 'with OAuth' do
+      #       # ...
+      #     end
+      #   end
+      class FeatureMethods < Cop
+        MSG = 'Use `%s` instead of `%s`.'.freeze
+
+        # https://git.io/v7rLu
```

Comment:
If you hit `y` on your keyboard when visiting this link then github will give you a link to a sha that will never break. https://help.github.com/articles/getting-permanent-links-to-files/